packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -125,7 +125,19 @@ const RootStack: React.FC = () => {
       })
     }
 
-    if (agent && state.deepLink.activeDeepLink && state.authentication.didAuthenticate) {
+    if (agent && agent.isInitialized && state.deepLink.activeDeepLink && state.authentication.didAuthenticate) {
```

Comment:
Able to add a little more description to this check. It may difficult to remember why this check is here 6mo down the road or by a new dev.

Reply:
all good, I actually did some refactoring which allowed me to remove the check